src/components/Feedback/FeedbackModal.tsx

```diff
@@ -153,7 +153,7 @@ const FeedbackModal = memo(({ user }: FeedbackModalProps) => {
             modalTitle={intl.formatMessage(messages.informRedhatDirection)}
             modalDescription={
               <Text>
-                {intl.formatMessage(messages.informDirectionDescription)}
+                {intl.formatMessage(messages.informDirectionDescription)}{' '}
```

Comment:
Although the empty string will work, it is very vulnerable to being deleted as this exact code is sometimes incorrectly injected by code formatters. It will be safer to use the HTML character code for a space: `&nbsp;`. We will never remove it.